packages/graphql/tests/tck/tck-test-files/connections/projections/update.test.ts

```diff
@@ -79,6 +79,8 @@ describe("Cypher -> Connections -> Projections -> Update", () => {
             "MATCH (this:Movie)
             WHERE this.title = $this_title
             WITH this
+            WITH this
```

Comment:
Having 2 consecutive `WITH this` is ok?
I feel we should try to avoid that, if its easy to do so

Reply:
This is an edge case that needs to be there, it wont effect the query in any way and is documented here https://github.com/neo4j/graphql/blob/master/packages/graphql/src/translate/translate-update.ts#L391